_includes/guide/try_the_api.html

```diff
@@ -44,15 +44,14 @@ <h3>
 </div>
 
 <h3>
-	BCDA Partially Adjudicated Enhancement  - Two Sizes
+  BCDA Partially Adjudicated Claims
```

Comment:
Suggest the following to be consistent with the titles of the datasets above
```suggestion
  Partially Adjudicated Claims Datasets - Two Sizes
```

Reply:
pls refer to the approved doc from CMS
https://confluence.cms.gov/x/2oJUMQ